DNSecure/Models/Presets.swift

```diff
@@ -115,5 +154,89 @@ enum Presets {
                 )
             )
         ),
+        .init(
+            name: "AdGuard DNS",
+            configuration: .dnsOverTLS(
+                DoTConfiguration(
+                    servers: [
+                        "94.140.14.140",
+                        "94.140.14.141",
+                        "2a10:50c0::1:ff",
+                        "2a10:50c0::2:ff",
+                    ],
+                    serverName: "dns-unfiltered.adguard.com"
+                )
+            )
+        ),
+        .init(
+            name: "AdGuard DNS",
+            configuration: .dnsOverHTTPS(
+                DoHConfiguration(
+                    servers: [
+                        "94.140.14.140",
+                        "94.140.14.141",
+                        "2a10:50c0::1:ff",
+                        "2a10:50c0::2:ff",
+                    ],
+                    serverURL: URL(string: "https://dns-unfiltered.adguard.com/dns-query")
+                )
+            )
+        ),
+        .init(
+            name: "AdGuard DNS (Block Ads / Trackers)",
+            configuration: .dnsOverTLS(
+                DoTConfiguration(
+                    servers: [
+                        "94.140.14.14",
+                        "94.140.15.15",
+                        "2a10:50c0::ad1:ff",
+                        "2a10:50c0::ad2:ff",
+                    ],
+                    serverName: "dns.adguard.com"
+                )
+            )
+        ),
+        .init(
+            name: "AdGuard DNS (Block Ads / Trackers)",
+            configuration: .dnsOverHTTPS(
+                DoHConfiguration(
+                    servers: [
+                        "94.140.14.14",
+                        "94.140.15.15",
+                        "2a10:50c0::ad1:ff",
+                        "2a10:50c0::ad2:ff",
+                    ],
+                    serverURL: URL(string: "https://dns.adguard.com/dns-query")
+                )
+            )
+        ),
+        .init(
+            name: "NextDNS",
+            configuration: .dnsOverTLS(
+                DoTConfiguration(
+                    servers: [
+                        "45.90.28.202",
+                        "45.90.30.202",
+                        "2a07:a8c0::",
+                        "2a07:a8c1::",
+                    ],
+                    serverName: "dns.nextdns.io"
+                )
+            )
+        ),
+        .init(
+            name: "NextDNS",
+            configuration: .dnsOverHTTPS(
+                DoHConfiguration(
+                    servers: [
+                        "45.90.28.202",
+                        "45.90.30.202",
+                        "2a07:a8c0::",
+                        "2a07:a8c1::",
+                    ],
+                    serverURL: URL(string: "https://dns.nextdns.io")
```

Comment:
NextDNS endpoints and IP addresses differ from person to person, so we shouldn't add them to the presets. What do you think?

Reply:
I was unsure about this. On the one hand, without an ID, I think it works as an unfiltered DNS. On the other hand, it was not meant to be used like that. I removed it.